app/pages/project/snapshots/SnapshotsPage.tsx

```diff
@@ -52,12 +54,12 @@ const EmptyState = () => (
   />
 )
 
+const snapshotList = (project: string) => getListQFn('snapshotList', { query: { project } })
```

Comment:
We can now share the query options between the prefetch in the loader and the query table, and we don't  have to specify `PAGE_SIZE` manually here to make sure it matches the query inside QueryTable.